webhook-service/lib/webhook_config.go

```diff
@@ -125,6 +131,9 @@ func verifyBeta1Request(request Request) error {
 	if !isMethodSupported(request.Method) {
 		return fmt.Errorf(webhookConfInvalid + "unsupported webhook request method")
 	}
+	if (request.User.Key != "" && request.User.Value == "") || (request.User.Key == "" && request.User.Value != "") {
```

Comment:
If one of them is declared but the other one is empty vice-versa, throw an error.